<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keithley Remote Control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="L_v_2">
        <property name="styleSheet">
         <string notr="true">font: 75 28pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Voltage:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="LCD_voltage">
        <property name="minimumSize">
         <size>
          <width>381</width>
          <height>109</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 64, 0);
border-color:rgb(170, 0, 0);
background-color:rgb(221, 232, 2);
font: italic bold 10pt &quot;Monotype Corsiva&quot;;</string>
        </property>
        <property name="digitCount">
         <number>10</number>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="L_voltage">
        <property name="styleSheet">
         <string notr="true">font: 75 28pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string> </string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="L_v">
        <property name="styleSheet">
         <string notr="true">font: 75 28pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>V</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="L_a_2">
        <property name="styleSheet">
         <string notr="true">font: 75 28pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>Current:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="LCD_current">
        <property name="minimumSize">
         <size>
          <width>381</width>
          <height>109</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 64, 0);
border-color:rgb(170, 0, 0);
background-color:rgb(221, 232, 2);
font: italic bold 10pt &quot;Monotype Corsiva&quot;;</string>
        </property>
        <property name="digitCount">
         <number>10</number>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="L_current">
        <property name="styleSheet">
         <string notr="true">font: 75 28pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string> </string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="L_a">
        <property name="styleSheet">
         <string notr="true">font: 75 28pt &quot;Times New Roman&quot;;</string>
        </property>
        <property name="text">
         <string>A</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionSerial"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="actionModel"/>
   </widget>
   <widget class="QMenu" name="menuFunction">
    <property name="title">
     <string>Function</string>
    </property>
    <addaction name="actionBias_Set"/>
    <addaction name="actionSwap"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetup"/>
   <addaction name="menuFunction"/>
   <addaction name="menuConnect"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionModel">
   <property name="text">
    <string>Model</string>
   </property>
  </action>
  <action name="actionSerial">
   <property name="text">
    <string>Serial</string>
   </property>
  </action>
  <action name="actionBias_Set">
   <property name="text">
    <string>Bias Set</string>
   </property>
  </action>
  <action name="actionSwap">
   <property name="text">
    <string>Sweep</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
